frontend/src/components/static_pages/about.css

```diff
@@ -6,17 +6,23 @@
 
 .images {
   margin: 0;
-  margin-bottom: 40px;
+  margin-bottom: 140px;
 }
 
 .images img {
   width: 400px;
+  height: 400px;
 }
 
 .sub-heading {
   font-size: 24px;
 }
 
+.people-heading {
+  font-size: 36pt;
```

Comment:
lets use `48px` here. Its the same as `36pt` in terms of size but `px` is the best practices as `pt` is more of a print measurement.